docker-compose.yml

```diff
@@ -142,6 +142,8 @@ services:
       - BACKSPLASH_ENABLE_GDAL=true
       - BACKSPLASH_CORE_STREAM_CONCURRENCY=16
       - BACKSPLASH_ENABLE_REQUEST_METRICS=false
+      - DATA_BUCKET
```

Comment:
Unless this is already in the ECS task definition, these environment variables will need to be added